src/Utilities/ProcessExtensions.cs

```diff
@@ -77,10 +77,16 @@ private static void GetAllChildIdsUnix(int parentId, ISet<int> children)
 
         private static void KillProcessUnix(int processId)
         {
-            RunProcessAndWaitForExit(
-                "kill",
-                $"-TERM {processId}",
-                out string _);
+            try
+            {
+                using Process process = Process.GetProcessById(processId);
+                process.Kill();
```

Comment:
Still a little uncomfortable with this, but we'll see what others say.

Reply:
If you ask the kernel to kill one of your child processes, it should be able to do that. There are no expected cases where this won't work.